tests/functional/test_libraries.py

```diff
@@ -102,7 +102,6 @@ def test_sphinx(tmpdir, pyi_builder, data_dir):
     # Note that including the data_dir fixture copies files needed by this test.
     pyi_builder.test_script('pyi_lib_sphinx.py')
 
-@pytest.mark.xfail(reason='pkg_resources is not supported yet.')
```

Comment:
My pleasure! Thanks for the hard work on PyInstaller. The fact that just dropping in a few extra datas defines and pkg_resources works shows how well-designed the program is.